tests/test_fields.py

```diff
@@ -721,6 +721,21 @@ def test_disallow_blank_with_trim_whitespace(self):
             field.run_validation('   ')
         assert exc_info.value.detail == ['This field may not be blank.']
 
+    def test_allow_null_bytes(self):
+        field = serializers.CharField(allow_null_bytes=True)
+        for value in ('\0', 'foo\0', '\0foo', 'foo\0foo'):
+            field.run_validation('\0')
```

Comment:
btw, these tests are using the hardcoded value instead of the one provided in the for loop.